src/mantarray_desktop_app/server.py

```diff
@@ -344,6 +345,59 @@ def boot_up() -> Response:
     return response
 
 
+@flask_app.route("/get_recordings", methods=["GET"])
+def get_recordings() -> Response:
+    """Get list of recordings from root recordings directory.
+
+    Can be invoked by: curl http://localhost:4567/get_recordings
+    """
+    try:
+        recording_dir = _get_values_from_process_monitor()["config_settings"]["recording_directory"]
+    except KeyError:
+        return Response(status="400 No root recording directory was found")
+
+    recordings_list = [
+        {
+            "name": dir,
+            "creation_time": datetime.fromtimestamp(
+                os.stat(os.path.join(recording_dir, dir)).st_mtime
+            ).strftime("%m-%d-%Y %H:%M.%S"),
+        }
+        for dir in os.listdir(recording_dir)
+        if glob.glob(os.path.join(recording_dir, dir, "*.h5"), recursive=True)
+    ]
+    response_dict = {"recordings_list": recordings_list, "root_recording_path": recording_dir}
+
+    return Response(json.dumps(response_dict), mimetype="application/json")
+
+
+@flask_app.route("/start_data_analysis", methods=["POST"])
+def run_mag_finding_analysis() -> Response:
+    """Route recieves list of recording paths to run analysis on locally.
+
+    Can be invoked by: curl http://localhost:4567/start_data_analysis
+    """
+    try:
+        config_setting = _get_values_from_process_monitor()["config_settings"]
+        recording_rootdir = config_setting["recording_directory"]
+    except KeyError:
+        return Response(status="400 Root directories were not found")
+
+    recording_paths = [
+        os.path.join(recording_rootdir, dirname) for dirname in request.get_json()["selected_recordings"]
+    ]
+
+    queue_command_to_main(
+        {
+            "communication_type": "mag_finding_analysis",
+            "command": "start_mag_analysis",
+            "content": {"recordings": recording_paths},
```

Comment:
For these commands you don't need to wrap everything other than the comm type and command underneath a "content" key. Most of the commands coming from the server just look like
```
{ 
    "communication_type": "mag_finding_analysis",
    "command": "start_mag_analysis",
    "recordings": recording_paths
}
```

Reply:
cool, i'll change it. Initially did that in case we ever needed more info bc I was passing the output directory here, but now what you're saying makes more sense